MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailPresenter.swift

```diff
@@ -7,12 +7,15 @@ final class MovieDetailPresenter {
     private let useCase: MoviesUseCaseProtocol!
     private let router: AppRouter!
 
-    let movieId: Int!
+    var movieId: Int!
```

Comment:
If you're using `setMovieId` function, then make this `private(set)` to indicate the variable is immutable from the outside.